lib/mrsk/configuration/role.rb

```diff
@@ -75,8 +90,6 @@ def traefik_labels
       if running_traefik?
         {
           "traefik.http.routers.#{traefik_service}.rule" => "PathPrefix(`/`)",
-          "traefik.http.services.#{traefik_service}.loadbalancer.healthcheck.path" => config.healthcheck["path"],
-          "traefik.http.services.#{traefik_service}.loadbalancer.healthcheck.interval" => "1s",
```

Comment:
We don't need Traefik to healthcheck since Docker is doing it. Also this wouldn't work for any custom `cmd` anyway.